server/tests/models/tagArticle.spec.js

```diff
@@ -27,46 +21,83 @@ before(async () => {
   }
 });
 
-describe('Tag Article relationship', async () => {
+describe('Tag Article relationship', () => {
+  before(async () => {
+    try {
+      const newUserId = await getUserId('testarticle@email.com', 'testarticle2');
+      const newCategoryId = await getCategoryId('technology');
+      articleData = getArticleData(newArticle, { userId: newUserId, categoryId: newCategoryId });
+      article = await Article.create(articleData);
+    } catch (error) {
+      console.log(error);
+    }
+  });
+
   it('should tag an article', async () => {
-    const newUserId = await getUserId('testarticle@email.com', 'testarticle2');
-    const newCategoryId = await getCategoryId('technology');
-    articleData = getArticleData(newArticle, { userId: newUserId, categoryId: newCategoryId });
-    article = await Article.create(articleData);
-    articleId = article.id;
-    tagId = await getTagId('EPIC');
-    const tagArticles = await article.addTags(tagId);
-    const tagArticleId = tagArticles[0].dataValues.ArticleId;
-    const tagArticleTagId = tagArticles[0].dataValues.TagId;
-
-    expect(tagArticleId).to.be.equal(articleId);
-    expect(tagArticleTagId).to.be.equal(tagId);
+    try {
+      articleId = article.id;
+      tagId = await getTagId('EPIC');
+      const tagArticles = await article.addTags(tagId);
+      const tagArticleId = tagArticles[0].dataValues.ArticleId;
+      const tagArticleTagId = tagArticles[0].dataValues.TagId;
+
+      expect(tagArticleId).to.be.equal(articleId);
+      expect(tagArticleTagId).to.be.equal(tagId);
+    } catch (error) {
+      console.log(error);
+    }
   });
 
   it('should return article tag', async () => {
-    const tagId1 = await getTagId('EPIC');
-    await article.addTags(tagId1);
-    const tagId2 = await getTagId('Andela');
-    await article.addTags(tagId2);
-    const articleTags = await article.getTags();
-    const articleTagsId = articleTags.map(getTags => getTags.dataValues.id);
-    await expect(articleTagsId).to.be.an('array').that.does.include(tagId);
-    await expect(articleTagsId).to.be.an('array').that.does.include(tagId2);
+    try {
+      articleId = article.id;
+      const articles = await Article.findByPk(articleId);
+      const tags = ['EPIC', 'Andela'];
+      const tagsId = [];
+
+      const tagArticles = tags.map(async (tag) => {
+        tagId = await getTagId(tag);
+        tagsId.push(tagId);
+        await articles.addTags(tagId);
+      });
+
+      await Promise.all(tagArticles);
+
+      const articleTags = await articles.getTags();
+      const articleTagsId = articleTags.map(getTags => getTags.dataValues.id);
+
+      expect(articleTagsId).to.be.an('array').that.does.include(tagsId[0]);
```

Comment:
You should consider removing your assertion from `try{}` block to avoid false assertion being caught by it

Reply:
You have two commit message, You should consider squashing it

Reply:
use `git log --oneline` to get all the abbreviated `{hashcodes}` of your commits,
then use `git rebase --interactive {hashcode}` to select an option `s or squash` would squash